.idea/modules/ArTEMiS_gatling.iml

```diff
@@ -17,6 +17,54 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="module" module-name="ArTEMiS_main" />
     <orderEntry type="module" module-name="ArTEMiS_test" production-on-test="" />
+    <orderEntry type="module-library" scope="RUNTIME">
```

Comment:
please remove these changes in this file again, they are not related to the quiz statistics